src/payment/data/service.js

```diff
@@ -3,12 +3,68 @@ import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 
 import handleRequestError from './handleRequestError';
 import { transformResults } from './utils';
+import { isWaffleFlagEnabled } from '../../data/waffleFlags';
+import { WAFFLE_FLAGS } from './constants';
 
 ensureConfig([
   'ECOMMERCE_BASE_URL',
   'LMS_BASE_URL',
 ], 'payment API service');
 
+/* eslint-disable no-use-before-define */ /* Chicken vs Egg...
+    but it is preferred the interpreter pickup errors rather than use magic strings */
```

Comment:
Might be worth elaborating what is the chicken vs. egg problem with `.name`.

Reply:
I assumed general knowlege of our tools, ill simplify.